docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       dockerfile: Dockerfile.dev
     environment:
     - APP_CONFIG_PATH=/data/config/config.json
-    - USE_EMBEDDED_DB
+    - USE_EMBEDDED_DB=1 #DON'T USE FOR PRODUCTION
```

Comment:
Please add a space between `#` and the rest of the comment to improve readability.